### ISSUE TITLE
GPU Memory size

### ISSUE THREAD

**Issue by raymondzmc** [microsoft/KEAR#6]
In the paper you mentioned "The batch size is set to 48 or smaller to fit the batch on to a single GPU."

I running your code with DeBERTa-Large on a single Nvidia V100 (16GB RAM) GPU, and even by breaking the input into 5 chunks (1 inference per question-choice score), I am unable to train the model without running into memory issues.

Could you provide some details regarding the hardware used?

**Comment by xycforgithub** [comment]
Hi raymondzmc,
We run our code with 8 V100 GPUs with 32GB memory. We used deepspeed to run DeBERTa jobs.

**Comment by raymondzmc** [comment]
Oh, I see, sadly I only have access to a single V100 with 16GB VRAM :(

BTW have this work been submitted to any conference? Would love to cite your work for an interpretation project that I'm working on.

**Comment by xycforgithub** [comment]
Yes it is in submission; we will update the citation once it is accepted. For now, you can cite the arxiv version. Thank you for your interest!
